portal-web/test/functional/com/liferay/portalweb/macros/Announcements.macro

```diff
@@ -0,0 +1,37 @@
+definition {
+
+	macro accessAddAnnouncements {
+		Click(locator1 = "Icon#PLUS");
+	}
+
+	macro addAnnouncement {
+		Type(
+			locator1 = "TextInput#TITLE",
+			value1 = "Announcement Title");
+
+		CKEditor.addContentInWebContent(
+			content = "Web Content Content",
+			fieldLabel = "Content");
```

Comment:
Does CKEditor.addContent work? Even if this macro works the name actually refers to Web Content which is a different feature altogether, so we should try to stay away from this